### PR TITLE
README.md: Link to Linux installation instructions

### PR DESCRIPTION
**Pull request by sjackman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
 ```
 
+Install Homebrew on [Linux and Windows 10 Subsystem for Linux](https://docs.brew.sh/Linuxbrew).
```
Comment (MikeMcQuaid) [inline README.md]: Maybe make this an H3 for added prominence?

```markdown
### Linux and Windows 10 Subsystem for Linux
```

and then the one-liner instructions?

Comment (sjackman) [reply README.md]: How do you feel about doing the same for https://brew.sh?

Comment (MikeMcQuaid) [reply README.md]: I've rather leave it as-is given the two-column thing it's got going. I did just think, though, that `Paste that in a Terminal prompt.` could be replaced with `Paste that in a macOS Terminal prompt.` for added clarity

Comment (sjackman) [reply README.md]: Updated this PR with H3.

Comment (sjackman) [reply README.md]: > could be replaced with Paste that in a macOS Terminal prompt.

Good idea. Done.

**Comment by sjackman** [comment]
Thanks for the review and merging, Mike.

**Pull request merged**